screens/aqi_screen/aqiPage.tsx

```diff
@@ -0,0 +1,210 @@
+import React, { FC } from "react";
+import { Text, View, StyleSheet, ImageBackground, Pressable } from "react-native";
+
+import { Ionicons, MaterialIcons } from "@expo/vector-icons";
+import { NativeStackScreenProps } from '@react-navigation/native-stack';
+import { StackParamList } from '../../App';
+
+type AQIScreenProps = NativeStackScreenProps<StackParamList, "Main">;
```

Comment:
Nice job inferring the type here.  I'm glad they got around to adding TS bindings to `react-navigation`.